### ISSUE TITLE
Point in Time Restore is not enabled when `should_copy_oplogs` is set to `true`, when copying backups to other regions

### ISSUE THREAD

**Issue by ericdalling** [mongodb/terraform-provider-mongodbatlas#1134]
### Terraform CLI and Terraform MongoDB Atlas Provider Version

```hcl
Terraform v1.3.7
MongoDB Atlas Provider v.1.8.2
```

### Terraform Configuration File 

```hcl
resource "mongodbatlas_cloud_backup_schedule" "cluster_backup_schedule" {
  project_id   = var.mongodb_project_id
  cluster_name = var.mongodb_cluster_name

  copy_settings {
    cloud_provider = "AWS"
    frequencies = [
      "HOURLY",
      "DAILY",
      "WEEKLY",
      "MONTHLY",
      "ON_DEMAND",
    ]
    region_name         = "US_EAST_1"
    replication_spec_id = module.cluster.replication_specs.*.id[0]
    should_copy_oplogs  = true
  }
}
```

### Steps to Reproduce

1. `terraform init`
2. `terraform apply`

### Expected Behavior

After running apply, the backup schedule should be modified to copy the back to a 2nd region and enable Point In Time Restore, because we have set the `should_copy_oplogs = true` in the `copy_settings` block.

### Actual Behavior

The `copy to another region` has been enabled for the desired snapshots, but Point In Time Restore say `No`, when it should be `Yes`

### Additional Context

![image](https://user-images.githubusercontent.com/326575/233499070-e1f4066f-dede-42cc-9e06-0ac7cab613e9.png)

### References

Documentation for the `should_copy_oplogs`: https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/cloud_backup_schedule#should_copy_oplogs

**Comment by Zuhairahmed** [comment]
Thanks @ericdalling we are investing and will revert back shortly

**Comment by martinstibbe** [comment]
@ericdalling This is a valid bug will be fixed in internal engineering ticket [INTMDB-783](https://jira.mongodb.org/browse/INTMDB-783)

**Comment by ericdalling** [comment]
Thanks for the update! I recommend updating your docs for the `copy_settings` block as well. It only shows up in an example but not in the Argument Reference section.

**Comment by Zuhairahmed** [comment]
hi @ericdalling we have released fix to this issue as part of v1.10.0 release. can you test and let us know if this issue has been resolved?

**Comment by Zuhairahmed** [comment]
Closing issue since this has been resolved and feel free to open a new issue in case you need anything else. thank you.
